libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -52,10 +52,16 @@ + (NSString *)localizedString:(NSString *)localizationKey
 
 + (UIImage *)imageNamed:(NSString *)name
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)
```

Comment:
The only drawbacks of using "resources" is as mentioned here: 

http://guides.cocoapods.org/syntax/podspec.html#resources

The recommended way is to use resource bundles. I have tried using resources and it works, but we run the risk of running into name collisions.